src/bp/ui-studio/src/web/components/Layout/Toolbar/ConfigForm.tsx

```diff
@@ -0,0 +1,249 @@
+    }
+  ]
+
+  if (licensing?.isPro) {
+    fields.push(
+      {
+        type: 'select',
+        key: 'defaultLanguage',
+        label: 'config.defaultLanguage',
+        options: languages
+      },
+      {
+        type: 'multi-select',
+        key: 'selectedLanguages',
+        label: 'languages',
+        options: languages
+      }
+    )
+  } else {
+    fields.push({
+      type: 'select',
+      key: 'defaultLanguage',
+      label: 'config.language',
+      options: languages
+    })
+  }
+
+  const advancedSettings: FormField[] = [
+    { type: 'text', key: 'website', label: 'config.website', placeholder: 'optional' },
+    { type: 'text', key: 'phoneNumber', label: 'config.phoneNumber', placeholder: 'optional' },
+    { type: 'text', key: 'emailAddress', label: 'config.contactEmail', placeholder: 'optional' },
+    { type: 'text', key: 'termsConditions', label: 'config.linkToTerms', placeholder: 'optional' },
+    { type: 'text', key: 'privacyPolicy', label: 'config.linkToPolicy', placeholder: 'optional' }
+  ]
+
+  const handleSave = async data => {
+    const botData: Partial<BotConfig> = {
+      name: data.name || '',
+      disabled: data.disabled,
+      private: data.private,
+      description: data.description || '',
+      defaultLanguage: data.defaultLanguage,
+      languages: data.selectedLanguages.map(x => x.value),
+      details: {
+        website: data.website || '',
+        phoneNumber: data.phoneNumber || '',
+        emailAddress: data.emailAddress || '',
+        termsConditions: data.termsConditions || '',
+        privacyPolicy: data.privacyPolicy || '',
+        avatarUrl: data.avatarUrl || '',
+        coverPictureUrl: data.coverPictureUrl || ''
+      }
+    }
+
+    const validation = Joi.validate(botData, BotEditSchema)
+    if (validation.error) {
+      setError(validation.error.details.reduce((acc, error) => ({ ...acc, [error.context.key]: error.message }), {}))
+      return
+    }
+
+    try {
+      const disableChanged = botData.disabled !== initialDisabledState.current
+      let allow = true
+
+      if (disableChanged && botData.disabled) {
+        allow = await confirmDialog(lang.tr('config.confirmUnmount'), {
+          acceptLabel: 'Unmount'
+        })
+      }
+
+      if (allow) {
+        await axios.post(`admin/bots/${bot.id}`, botData, axiosConfig)
+        setError(undefined)
+
+        if (disableChanged) {
+          window.location.reload()
+        } else {
+          fetchBotInformation()
+        }
+      } else {
+        setError(undefined)
+      }
+    } catch (err) {
+      setError(err.response?.data)
+    }
+  }
+
+  return (
+    <RightSidebar className={sharedStyle.wrapper} canOutsideClickClose close={() => close()}>
+      <div className={sharedStyle.formHeader}>
+        <Tabs id="configurationFormTabs">
+          <Tab id="configuration" title={lang.tr('toolbar.configuration')} />
+        </Tabs>
+      </div>
+
+      {!!languages.length && (
```

Comment:
It's possible that there is no language if the user doesn't use NLU or if the language server is not available, but it shouldnt prevent the use of the config form

Reply:
Ok, I added a flag instead, I had issues with the selected langs not showing up if the form was rendered before the options were loaded